confd/pkg/resource/template/fileStat_posix.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
Are these just different syntaxes for different go build versions? What's up with this change?

Reply:
hmmm, you know i just assumed this was a go 1.17 thing like the second require block.  I explicitly checked the latter ... but i'll fess up i didn't check the build tag thing..   I'll just double check it is ok.

Reply:
yeah syntax has changed in 1.17 .... I guess make fix keeps the old one and adds the new one as a safety measure.